Doc/Manual/SWIG.html

```diff
@@ -2263,6 +2278,7 @@ <H4><a name="SWIG_chosen_unignore">5.4.7.5 Ignoring everything then wrapping a f
 %rename($ignore, %$isclass) ""; // Only ignore all classes
```

Comment:
This existing example here is actually wrong - it needs to be `"$ignore"` (i.e. in quotes or else you get a parse error).

Reply:
OK - solved - I'd failed to put a name after `%module` in the line above in my `test.i` and SWIG reports the parse error in the line after because that's where the next token is.